packages/stream-model-handler/src/__tests__/model-handler.test.ts

```diff
@@ -127,20 +130,50 @@ const CONTENT_WITH_INVALID_SCHEMA = {
   },
 }
 
-const ThreeIdResolver = {
-  '3': async (did) => ({
+function resolutionResult(didUrl: string, optRotateDate?: string): DIDResolutionResult {
```

Comment:
I think this should import the new `resolutionResult` from `test-utils/src/index.ts` it's the same code